tests/generic_config_updater/test_dhcp_relay.py

```diff
@@ -242,177 +238,54 @@ def dhcp_severs_by_vlanid(duthost, vlanid):
 
     return output
 
+
 def expect_res_success_by_vlanid(duthost, vlanid, expected_content_list, unexpected_content_list):
     ensure_dhcp_server_up(duthost)
     output = dhcp_severs_by_vlanid(duthost, vlanid)
     expect_res_success(duthost, output, expected_content_list, unexpected_content_list)
 
-# DHCP_RELAY TEST
-def test_dhcp_relay_tc1_apply_empty(duthost, init_dhcp_server_config):
```

Comment:
>test_dhcp_relay_tc1_apply_empty [](http://example.com/codeflow?start=4&length=31)

I see several testcases removed, why?

Reply:
Previous we have two setup(CONFIG_CLEANUP, CONFIG_ADD_DEFAULT). 
- However, there is only one test running in [CONFIG_CLEANUP](https://github.com/Azure/sonic-mgmt/pull/5866/commits/c16aab3a10cc89b1f79132001e1752f6d12cd694#diff-2898f8c40b20596114b8832da87b24d0c19c8fa2cd6b8380b8a0fa15cdb12de0L269). That case is about remove non-existed server and could be covered in CONFIG_ADD_DEFAULT.
- test_dhcp_relay_tc1_apply_empty is to apply empty json_patch. This is also covered in [conftest.py.](https://github.com/Azure/sonic-mgmt/blob/master/tests/generic_config_updater/conftest.py#L61)
- Another optimization is to group add and remove server into one testcase. It optimize the verification of two operation at one test.